src/include/rbd_types.h

```diff
@@ -77,6 +77,7 @@
 #define RBD_GROUP_HEADER_PREFIX "rbd_group_header."
 
 #define RBD_GROUP_DIRECTORY "rbd_group_directory"
+#define RBD_GROUP_UPLINK "rbd_group_uplink"
```

Comment:
I would like to use this constant in test_cls_rbd.cc. The test won't see it in cls_rbd.cc. Is there any other appropriate header file for this constant?